src/lib/QtWidgets/QDateEdit.ts

```diff
@@ -25,13 +25,13 @@ export class QDateEdit extends QDateTimeEdit {
     constructor();
     constructor(parent: NodeWidget<any>);
     constructor(parent?: NodeWidget<any>) {
+        super();
         let native;
         if (parent) {
             native = new addon.QDateEdit(parent.native);
         } else {
             native = new addon.QDateEdit();
         }
-        super(native);
```

Comment:
Hi @feng8848  Great work on the pr! 

There is an issue with removing `super(native)`
Here we do this so that EventWidget gets the native instance and sets up event listeners needed for event handling. 
By doing this the event listeners wont work for qdateedit and qtimeedit.

Typescript doesnt show an error because the native widget is passed to eventwidget by the QDateTimeEdit 

The implementation of QDateEdit and QTimeEdit is wrong. Since initializing qdateedit will also initialise a native QDateTimeEdit.  

To solve this we will implement NodeDateTimeEdit and use that for inheritance. You can check my follow up commit on this Pr to see what I mean.

Reply:
This is the commit: https://github.com/nodegui/nodegui/pull/383/commits/6f6577732233f8f805cde3a46cbbb653d04f4ed1